spec-examples/src/test/java/org/beanvalidation/specexamples/constraintdeclarationvalidation/resolution/NonEmpty.java

```diff
@@ -1,8 +1,8 @@
 /*
- * Bean Validation: constrain once, validate everywhere.
+ * Jakarta Bean Validation: constrain once, validate everywhere.
  *
- * License: Apache License, Version 2.0
```

Comment:
@emmanuelbernard, so far I understood the spec and API sources would remain under Apache v2 license, see also the latest mail from @starksm64. But it still doesn't quite compute to me, should the sources be Apache v2 but the published JavaDoc EFSL, Scott?

Reply:
Yeah, that's what I asked on Zulip and apparently, Emmanuel agrees.

Reply:
Isn't it the exact same situation as before i.e. the final versions of spec and Javadoc has to be under the Eclipse license (they were using an Oracle one before)?

And the API is under ASL 2 and stays ASL 2, even in Final.

Reply:
I got confirmation from Scott that the previous situation is similar. Just put the EFSL in the final build of a spec.